aspnetcore/tutorials/choose-web-ui.md

```diff
@@ -0,0 +1,193 @@
+
+### ASP.NET Core Razor Pages
+
+Razor Pages is a page-based model. UI and business logic concerns are kept separate, but within the page. Razor Pages is the recommended way to create new page-based or form-based apps for developers new to ASP.NET Core. Razor Pages provides an easier starting point than ASP.NET Core MVC.
+
+Razor Pages benefits, in addition to the server rendering benefits:
+
+* Quickly build and update UI. Code for the page is kept with the page, while keeping UI and business logic concerns separate.
+* Testable and scales to large apps.
+* Keep your ASP.NET Core pages organized in a simpler way than ASP.NET MVC:
+  * View specific logic and view models can be kept together in their own namespace and directory.
+  * Groups of related pages can be kept in their own namespace and directory.
+
+To get started with your first ASP.NET Core Razor Pages app, see [Get started with Razor Pages](xref:tutorials/razor-pages/razor-pages-start). For a complete overview of ASP.NET Core Razor Pages, its architecture and benefits, see: [Introduction to Razor Pages in ASP.NET Core](xref:razor-pages/index)
+
+### ASP.NET Core MVC
+
+ASP.NET MVC renders UI on the server and uses a Model-View-Controller (MVC) architectural pattern. The MVC pattern separates an app into three main groups of components: Models, Views, and Controllers. User requests are routed to a controller. The controller is responsible for working with the model to perform user actions or retrieve results of queries. The controller chooses the view to display to the user, and provides it with any model data it requires. Support for Razor Pages is built on ASP.NET Core MVC.
+
+MVC benefits, in addition to the server rendering:
+
+* Some developers consider it easier to scale the web UI app in terms of complexity. They feel it's easier to code, debug, and test the model, view, or controller when it has a single job.
+* Clear [separation of concerns](/dotnet/standard/modern-web-apps-azure-architecture/architectural-principles#separation-of-concerns) for maximum flexibility.
+* The Model-View-Controller separation of responsibilities ensures that the business model can evolve without being tightly coupled to low-level implementation details.
+
+To get started with  ASP.NET Core MVC, see [Get started with MVC](xref:tutorials/first-mvc-app/start-mvc). For an overview of ASP.NET Core MVC's architecture and benefits, see [Overview of ASP.NET Core MVC](xref:mvc/overview)
+
+## Choose a predominantly client rendered ASP.NET Core solution
+
+The following section briefly explains the ASP.NET Core web UI client rendered models available and provides links to get started.
+
+### Blazor
+
+Blazor apps are composed of Blazor components: reusable, web UI components implemented using C#, HTML, and CSS. Both client and server code are written in C#, allowing sharing code and libraries. Blazor components can be rendered or pre-rendered from views and pages.
```

Comment:
Here is what we currently have on the top entry point of content on the marketing oriented site:
"Blazor apps are composed of reusable web UI components implemented using C#, HTML, and CSS. "

I will go this route however of explicitly calling them Razor components and you have a very good point Like.  If we decide to do something different later, we should change it throughout the docs.

Reply:
Here's the cross-ref for the definition in the Blazor Overview. IIRC, this was reviewed/approved by @danroth27, but that was perhaps years ago with nit mods since then.

https://docs.microsoft.com/en-us/aspnet/core/blazor/?view=aspnetcore-5.0#components

Reply:
... and I see that in my edit I left "element" there. I'm not sure I feed great about it (element ... HTML element ... overload). There are synonyms, but the rest of the sentence scopes it cleanly. I don't think there's any danger of confusion really.